.github/workflows/cowFi-tokens.yml

```diff
@@ -0,0 +1,100 @@
+name: 'Cron: cow.fi tokens list'
+
+on:
+  schedule:
+    - cron: '30 * * * *'
+
+# Required for authenticating with AWS IAM
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+
+      - name: Set output of cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
Deprecated way of setting output vars.
Also, you can cache yarn output in the setup node step.
See:

https://github.com/cowprotocol/cowswap/blob/fb24d411a5df88084117f4eb27fd0500db495851/.github/workflows/ci.yml#L29-L50

Reply:
https://github.com/cowprotocol/token-lists/pull/304